egs/swbd/asr1/RESULTS

```diff
@@ -4,4 +4,20 @@ exp/train_nodup_a01_pt_blstmp_e6/decode_eval2000_beam20_eacc.best_p0.1_len0.0-0.
 exp/train_nodup_a01_pt_blstmp_e6/decode_rt03_beam20_eacc.best_p0.1_len0.0-0.0/result.txt:|  SPKR         | # Snt    # Wrd  |  Corr     Sub      Del      Ins     Err    S.Err  |
 exp/train_nodup_a01_pt_blstmp_e6/decode_rt03_beam20_eacc.best_p0.1_len0.0-0.0/result.txt:|  Sum/Avg      | 8422    321545  |  78.2    10.6     11.2     12.3    34.1     79.6  |
 exp/train_nodup_a01_pt_blstmp_e6/decode_train_dev_beam20_eacc.best_p0.1_len0.0-0.0/result.txt:|  SPKR       |  # Snt     # Wrd  |  Corr       Sub      Del       Ins      Err     S.Err  |
-exp/train_nodup_a01_pt_blstmp_e6/decode_train_dev_beam20_eacc.best_p0.1_len0.0-0.0/result.txt:|  Sum/Avg    |  3999     235886  |  87.4       6.4      6.2       5.0     17.6      65.4  |
\ No newline at end of file
+exp/train_nodup_a01_pt_blstmp_e6/decode_train_dev_beam20_eacc.best_p0.1_len0.0-0.0/result.txt:|  Sum/Avg    |  3999     235886  |  87.4       6.4      6.2       5.0     17.6      65.4  |
+
+$ grep -e Avg -e SPKR -m 2 exp/train_nodup_pytorch_transformer_e12_BPE500_lr5_30epoch/decode_*_decode_pytorch_transformer_ctcweight0.3/scoring/hyp.*.sys
```

Comment:
We usually perform the transformer with 50-100 epochs.
Could you resume to train your model with such epochs?

Reply:
Okay, I have already tried using 30 epochs and 100 epochs.

| Model | WER/S.Err (overall) | WER/S.Err (callhm) | WER/S.Err (swbd) |
| ---- | ---- | ---- | ---- |
| Transformer (15 epochs) | **20.4/59.3** | **26.0/63.3** | 14.7/53.5 |
| Transformer (30 epochs) | 20.0/59.4 | 26.0/64.5 | **14.0/52.0** |
| Transformer (100 epochs) | 20.3/59.8 | 26.5/65.4 | 14.1/51.6 |
| RNN | 22.0/60.1 | 28.5/64.6 | 15.6/53.5 |